src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -21,8 +21,13 @@ export default function withAnnotations<P extends object>(
     class ComponentWithAnnotations extends React.Component<P & WithAnnotationsProps, AnnotatorState> {
         static displayName: string;
 
+        handleActiveChange = (annotationId: string | null): void => {
+            this.setState({ activeAnnotationId: annotationId });
+        };
+
         state: AnnotatorState = {
             activeAnnotationId: null,
+            setActiveAnnotationId: this.handleActiveChange,
```

Comment:
I think instead of directly setting it back to state, we may want to emit the `annotations_active_set` event back to the annotator, and let that naturally come back as a `annotations_active_change` event which will trigger `handleActiveChange`